cypress/support/pages/signIn.pageObject.js

```diff
@@ -29,6 +37,16 @@ class SignInPageObject extends PageObject {
     this.signInBtn
       .click();
   }
+
+  verifyEmailErorr() {
+    this.emailErrorMessage
+      .should('be.visible');
+  }
+
+  verifyPasswordErorr() {
```

Comment:
There is a typo in the method name `verifyPasswordErorr`. It should likely be `verifyPasswordError` to correctly reflect the purpose of verifying a password error.